backend/frontend/tests/test_card_start_button.py

```diff
@@ -0,0 +1,71 @@
+from core.tests.factories import UserFactory
+from curriculum_tracking.tests.factories import (
+    AgileCardFactory,
+    ContentItemFactory,
+    ProjectContentItemFactory,
+)
+from curriculum_tracking.models import ContentItem, AgileCard, RecruitProject
+from .frontend_test_mixin import FrontendTestMixin
+
+
+class TestCardStartButton(FrontendTestMixin):
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory()
+        self.user.set_password(self.user.email)
+        self.user.save()
+
+        self.do_login(self.user)
+
+    def make_topic_card(self):
+        self.card = AgileCardFactory(
+            content_item=ContentItemFactory(content_type=ContentItem.TOPIC),
+            status=AgileCard.READY,
+        )
+
+        self.card.assignees.set([self.user])
+
+    def make_project_card(self, project_submission_type):
+        self.card = AgileCardFactory(
+            content_item=ContentItemFactory(
+                content_type=ContentItem.PROJECT,
+                project_submission_type=project_submission_type,
+            ),
+            status=AgileCard.READY,
+        )
+
+        self.card.assignees.set([self.user])
+
+    def test_start_button_moves_topic_card_to_ip_column(self):
+        self.make_topic_card()
+
+        can_start = self.card.request_user_can_start(self.user)
+
+        if can_start:
```

Comment:
If the current user can't start the card then this test will always pass, and it will test nothing.

If the user cant start the card then it means the test wasn't set up properly and it should raise an error.